### PR TITLE
Updating packages to get to a stable point + fixing errors associated with update

### PR DESCRIPTION
**Pull request by ryelo**
This puts us on par with the "stable" patternfly packages. I tested with Remediations + Dashboard and saw no breaking changes. It seems like the only places that reference the old pf were a few places inside of chrome, because all of the other apps are using react.

A few other packages were updated due to a vulnerability.

### CODE REVIEW

**Comment by karelhala** [comment]
Looks like we need to update babelrc config as well.

**Comment by kruai** [comment]
When I start npm on this latest commit, I get a ton of traces in my output:
```
Trace: The node type SpreadProperty has been renamed to SpreadElement
    at Object.isSpreadProperty (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/types/lib/validators/generated/index.js:4272:11)
    at hasSpread (/home/aprice/workspace/insights-chrome/node_modules/babel-plugin-transform-object-rest-spread/lib/index.js:38:13)
    at PluginPass.ObjectExpression (/home/aprice/workspace/insights-chrome/node_modules/babel-plugin-transform-object-rest-spread/lib/index.js:234:14)
    at newFn (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/visitors.js:193:21)
    at NodePath._call (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/path/context.js:53:20)
    at NodePath.call (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/path/context.js:88:12)
    at TraversalContext.visitQueue (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/context.js:118:16)
    at TraversalContext.visitSingle (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/context.js:90:19)
    at TraversalContext.visit (/home/aprice/workspace/insights-chrome/node_modules/@babel/core/node_modules/@babel/traverse/lib/context.js:146:19)
```
The browser output looks fine, though. Not sure if those traces are just happening to me.

**Comment by ryelo** [comment]
@kruai That should only happen the first time you run `npm`

**Comment by kruai** [comment]
@ryelo You're right, it only happened once. Looks good.

**Comment by ryelo** [comment]
On hold until: https://github.com/RedHatInsights/insights-frontend-components/pull/335

File: package.json [hunk 1]
```diff
@@ -41,9 +41,13 @@
   },
   "homepage": "https://github.com/RedHatInsights/insights-chrome#readme",
   "devDependencies": {
+    "@babel/core": "^7.3.3",
```
Comment (karelhala) [inline package.json]: If we are using new `babel` can we also update all babel deps to version 7? Or create an issue to mark it and we should update in follow-up PR

Comment (ryelo) [reply package.json]: Let's create an issue to work on it in a follow-up

**Pull request merged**